database/seeder.js

```diff
@@ -87,25 +82,22 @@ thingsToDo.forEach((todo) => {
   });
 });
 
+offering.forEach((item) => {
+  db.query(insertOfferingSql, item, (err) => {
+    if (err) throw err;
+  });
+});
 
 
-const getSimilarListing = function(callback) {
-  db.query(`SELECT * FROM similar_listing`, callback);
-}
-
-const getThingsToDo = function(callback) {
-  db.query(`SELECT * FROM things_to_do`, callback);
-}
-
-
-
-module.exports = {
-  getSimilarListing,
-  getThingsToDo 
-};
-
-
-
-
+// const getSimilarListing = function(callback) {
```

Comment:
If Lines 92-103 are moved to the indexDB.js file, I believe we can remove them here completely.

Reply:
Done